src/dashboards/JCDecaux/Content/MobilityTiles/MobilityTiles.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { FormFactor } from '@entur/sdk/lib/mobility/types'
+import { MobilityTile } from './MobilityTile/MobilityTile'
+import './MobilityTiles.scss'
+
+const MobilityTiles = ({ numberOfBikes }: Props): JSX.Element | null => (
+    <div className="mobility-tiles-wrapper">
+        <MobilityTile
+            formFactor={FormFactor.CAR}
+            numberOfBikes={numberOfBikes}
+        />
+    </div>
+)
```

Comment:
"numberOfBikes" med FormFactor.CAR ser noe out of place ut, kan det forandres til numberOfVehicles f.eks.?

Reply:
Oui